### PR TITLE
More CEX addresses

### PR DESCRIPTION
**Pull request by hildobby**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/cex/ethereum/cex_ethereum_addresses.sql [hunk 1]
```diff
@@ -578,6 +615,9 @@ FROM (VALUES
     , ('ethereum', 0xb709d82f0706476457ae6bad7c3534fbf424382c, 'Panda Exchange', 'Panda Exchange Hot Wallet 2', 'hildobby', date '2022-08-28') -- https://etherscan.io/address/0xb709d82f0706476457ae6bad7c3534fbf424382c
     -- IDAX
     , ('ethereum', 0x3c11c3025ce387d76c2eddf1493ec55a8cc2a0f7, 'IDAX', 'IDAX', 'hildobby', date '2022-08-28') -- https://etherscan.io/address/0x3c11c3025ce387d76c2eddf1493ec55a8cc2a0f7
+    -- IDEX, Source: https://etherscan.io/accounts/label/IDEX
+    , ('ethereum', 0x2a0c0dbecc7e4d658f48e01e3fa353f44050c208, 'IDEX', 'IDEX 1', 'hildobby', date '2023-09-08') -
```
Comment (Hosuke) [inline models/cex/ethereum/cex_ethereum_addresses.sql:619]:
```suggestion
    , ('ethereum', 0x2a0c0dbecc7e4d658f48e01e3fa353f44050c208, 'IDEX', 'IDEX 1', 'hildobby', date '2023-09-08')
```

**Review by Hosuke — APPROVED** [review]

File: models/cex/ethereum/cex_ethereum_addresses.sql [hunk 2]
```diff
@@ -744,10 +846,27 @@ FROM (VALUES
...
+    , ('ethereum', 0x7ed1e469fcb3ee19c0366d829e291451be638e59, 'Freewallet', 'Freewallet 1', 'hildobby', date '2023-09-09')
     -- Phemex, source: https://phemex.com/proof-of-reserves
     , ('ethereum', 0xf7d13c7dbec85ff86ee815f6dcbb3dedac78ca49, 'Phemex', 'Phemex 1', 'hildobby', date '2023-04-06')
+    , ('ethereum', 0x50be13b54f3eebbe415d20250598d81280e56772, 'Phemex', 'Phemex 2', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0x50be13b54f3eebbe415d20250598d81280e56772, 'Phemex', 'Phemex 3', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0x35d2d03607b9155b42cf673102fe58251ac4f644, 'Phemex', 'Phemex 4', 'hildobby', date '2023-09-12')
     -- Cake DeFi, source: https://cakedefi.com/transparency
     , ('ethereum', 0x94fa70d079d76279e1815ce403e9b985bccc82ac, 'Cake DeFi', 'Cake DeFi 1', 'hildobby', date '2023-04-06')
+    -- EXMO
+    , ('ethereum', 0xd7b9a9b2f665849c4071ad5af77d8c76aa30fb32, 'EXMO', 'EXMO 1', 'hildobby', date '2023-09-12')
```
Comment (Hosuke) [inline models/cex/ethereum/cex_ethereum_addresses.sql:865]: Duped line:
https://github.com/duneanalytics/spellbook/blob/3fe30ef1aaa61965227a2abcd7ca3cb6b35b0ff0/models/cex/ethereum/cex_ethereum_addresses.sql#L471

File: models/cex/ethereum/cex_ethereum_addresses.sql [hunk 3]
```diff
@@ -357,6 +388,10 @@ FROM (VALUES
     , ('ethereum', 0x8fa8af91c675452200e49b4683a33ca2e1a34e42, 'Bithumb', 'Bithumb Old Address 3', 'hildobby', date '2022-08-28')
     , ('ethereum', 0x3b83cd1a8e516b6eb9f1af992e9354b15a6f9672, 'Bithumb', 'Bithumb Old Address 4', 'hildobby', date '2022-08-28')
     , ('ethereum', 0x03599a2429871e6be1b154fb9c24691f9d301865, 'Bithumb', 'Bithumb Deposit Funder', 'hildobby', date '2023-08-31')
+    -- Bitso
+    , ('ethereum', 0x58b704065b7aff3ed351052f8560019e05925023, 'Bitso', 'Bitso 1', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0x29d5527caa78f1946a409fa6acaf14a0a4a0274b, 'Bitso', 'Bitso 2', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0x58b704065b7aff3ed351052f8560019e05925023, 'Bitso', 'Bitso 3', 'hildobby', date '2023-09-12')
```
Comment (Hosuke) [inline models/cex/ethereum/cex_ethereum_addresses.sql:394]: Duped line:
https://github.com/duneanalytics/spellbook/blob/3fe30ef1aaa61965227a2abcd7ca3cb6b35b0ff0/models/cex/ethereum/cex_ethereum_addresses.sql#L392

File: models/cex/ethereum/cex_ethereum_addresses.sql [hunk 4]
```diff
@@ -744,10 +846,27 @@ FROM (VALUES
...
+    -- Cobo
+    , ('ethereum', 0xa9c7d31bb1879bff8be25ead2f59b310a52b7c5a, 'Cobo', 'Cobo 1', 'hildobby', date '2023-09-09')
+    -- Cryptonator
+    , ('ethereum', 0x4FED1fC4144c223aE3C1553be203cDFcbD38C581, 'Cryptonator', 'Cryptonator 1', 'hildobby', date '2023-09-09')
+    -- Freewallet
+    , ('ethereum', 0x7ed1e469fcb3ee19c0366d829e291451be638e59, 'Freewallet', 'Freewallet 1', 'hildobby', date '2023-09-09')
     -- Phemex, source: https://phemex.com/proof-of-reserves
     , ('ethereum', 0xf7d13c7dbec85ff86ee815f6dcbb3dedac78ca49, 'Phemex', 'Phemex 1', 'hildobby', date '2023-04-06')
+    , ('ethereum', 0x50be13b54f3eebbe415d20250598d81280e56772, 'Phemex', 'Phemex 2', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0x50be13b54f3eebbe415d20250598d81280e56772, 'Phemex', 'Phemex 3', 'hildobby', date '2023-09-12')
```
Comment (Hosuke) [inline models/cex/ethereum/cex_ethereum_addresses.sql:860]: These 2 lines are duplicated.

File: models/cex/ethereum/cex_ethereum_addresses.sql [hunk 5]
```diff
@@ -744,10 +846,27 @@ FROM (VALUES
...
+    , ('ethereum', 0x35d2d03607b9155b42cf673102fe58251ac4f644, 'Phemex', 'Phemex 4', 'hildobby', date '2023-09-12')
     -- Cake DeFi, source: https://cakedefi.com/transparency
     , ('ethereum', 0x94fa70d079d76279e1815ce403e9b985bccc82ac, 'Cake DeFi', 'Cake DeFi 1', 'hildobby', date '2023-04-06')
+    -- EXMO
+    , ('ethereum', 0xd7b9a9b2f665849c4071ad5af77d8c76aa30fb32, 'EXMO', 'EXMO 1', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0xc179fbddc946694d11185d4e15dbba5fd0adac0a, 'EXMO', 'EXMO 2', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0xb36cbe7f95a39984384e6aa4068b02c1697ef80e, 'EXMO', 'EXMO 3', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0xfff52a282c215aed0c4c71786d85519bbcc86464, 'EXMO', 'EXMO 4', 'hildobby', date '2023-09-12')
+    , ('ethereum', 0x1fd6267f0d86f62d88172b998390afee2a1f54b6, 'EXMO', 'EXMO 5', 'hildobby', date '2023-09-12')
```
Comment (Hosuke) [inline models/cex/ethereum/cex_ethereum_addresses.sql:869]: Duped line:
https://github.com/duneanalytics/spellbook/blob/3fe30ef1aaa61965227a2abcd7ca3cb6b35b0ff0/models/cex/ethereum/cex_ethereum_addresses.sql#L470

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/cex/ethereum/cex_ethereum_addresses.sql (hunk 5 above)
Comment (hildobby) [reply models/cex/ethereum/cex_ethereum_addresses.sql:872]: fixed

File: models/cex/ethereum/cex_ethereum_addresses.sql (hunk 4 above)
Comment (hildobby) [reply models/cex/ethereum/cex_ethereum_addresses.sql]: fixed

File: models/cex/ethereum/cex_ethereum_addresses.sql (hunk 3 above)
Comment (hildobby) [reply models/cex/ethereum/cex_ethereum_addresses.sql]: fixed

File: models/cex/ethereum/cex_ethereum_addresses.sql (hunk 2 above)
Comment (hildobby) [reply models/cex/ethereum/cex_ethereum_addresses.sql:868]: fixed

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @hildobby

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
since wash trades use this downstream, let's wait to merge this with that PR (one outstanding minor question there)

**Pull request merged**